Source/Mdk.CommandLine/IngameScript/Pack/DefaultProcessors/PreprocessorConditionals.cs

```diff
@@ -69,6 +69,8 @@ public async Task<Document> ProcessAsync(Document document, IPackContext context
                 //since "#define " is 8 in length
```

Comment:
You don't need to parse the value yourself. Take a look at how the #if is parsed. It's already parsed for you, you only need to take the first token and sanity-check it.